packages/core-flows/src/stock-location/steps/update-stock-locations.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  IStockLocationServiceNext,
+  UpdateStockLocationNextInput,
+} from "@medusajs/types"
+import { StepResponse, createStep } from "@medusajs/workflows-sdk"
+import {
+  convertItemResponseToUpdateRequest,
+  getSelectsAndRelationsFromObjectArray,
+} from "@medusajs/utils"
+
+import { ModuleRegistrationName } from "@medusajs/modules-sdk"
+
+export const updateStockLocationsStepId = "update-stock-locations-step"
+export const updateStockLocationsStep = createStep(
```

Comment:
todo: As per our convention, this should use the input type `{ selector, update }` instead of relying on the ID for the updates. You can reference the `update-regions` or `update-users` steps.